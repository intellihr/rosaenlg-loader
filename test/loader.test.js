import {
  compile,
  execute,
  getCompiler,
  getErrors,
  getWarnings,
  readAsset,
} from './helpers';

describe('loader', () => {
  it('should work for simple case', async () => {
    const compiler = getCompiler('simple.js', {
      language: 'en_US'
    });
    const stats = await compile(compiler);

    expect(
      execute(readAsset('main.bundle.js', compiler, stats))
    ).toMatchSnapshot('result');
    expect(getErrors(stats)).toMatchSnapshot('errors');
    expect(getWarnings(stats)).toMatchSnapshot('warnings');
  });
});
