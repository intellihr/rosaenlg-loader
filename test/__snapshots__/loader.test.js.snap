// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`loader should work for simple case: errors 1`] = `Array []`;

exports[`loader should work for simple case: result 1`] = `"Simple test"`;

exports[`loader should work for simple case: warnings 1`] = `Array []`;
